.github/workflows/ci.yml

```diff
@@ -308,7 +308,7 @@ jobs:
           # We decided to exclude all external HTTP requests but the ones that under the domain greatexpectations.io
           # The reason is to avoid having network errors such as pages that throw 429 after too many requests (like Github)
           # and to prevent other possible errors related to user agent or lychee capturing hrefs from metadata that don't resolve to a specific page (preconnects in JS)
-          args: "--exclude='http.*' --include='^https://(.+\\.)?greatexpectations\\.io/' 'docs/docusaurus/build/**/*.html'"
```

Comment:
This is a workaround due to limitations of the new link checker. It doesn't know how to check new pages. I will put in a follow-up PR reverting this one line change.